Cabal/doc/nix-local-build.rst

```diff
@@ -467,6 +467,13 @@ its ``cabal`` executable:
 For libraries and local packages see
 `Unsupported commands <#unsupported-commands>`__
 
+cabal new-clean
+---------------
+
+``cabal new-clean [FLAGS]`` cleans up the temporary files and build artifacts
+stored in the ``dist-newstyle`` folder. By default, it removes the entire folder,
+but it can also selectively remove the build artifacts and spare the caches within.
```

Comment:
I guess this is no longer true? Or does this mean `--save-config`? In any case, the wording is a bit vague.

Reply:
Also please document the distPref and projectFile flags here.